File: learn/linkedlist/ll.py
class Node:
    def __init__(self , data) -> None:
        self.data = data
        self.next = None

class LinkedList:
    def __init__(self,head=None) -> None:
        self.head = head

    def display(self):
        p = self.head

        while p != None:
            print( p.data )
            p = p.next 

    def recursive_display(self):

        p = self.head
        if p.next == None:
            

File: learn/linkedlist/LL.java
public class LL{
    public class Node{
        int data;
        Node next;
    } 

    public static void display( Node p ){
        while ( p != null ){
            System.out.println( p.data );
            p = p.next;
        }
    }

    public static void main(String[] args) {
        
        
        System.out.println(1+1);   
	System.out.println("Linked-List");
    }
}
